src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java

```diff
@@ -403,6 +403,9 @@ private void setupHosting(String channel, String sender, String hostingId, Strin
                     + "* [Accept the invitation to the Jenkins CI Org on Github|https://github.com/jenkinsci]\n"
                     + "* [Request upload permission|https://wiki.jenkins-ci.org/display/JENKINS/Hosting+Plugins#HostingPlugins-Requestuploadpermissions]\n"
                     + "* [Releasing your plugin|https://wiki.jenkins-ci.org/display/JENKINS/Hosting+Plugins#HostingPlugins-Releasingtojenkins-ci.org]\n"
+                    + "\n\nIn order for your plugin to be built by the Jenkins CI infrastructure and check PR's, please add a [Jenkinsfile|https://jenkins.io/doc/book/pipeline/jenkinsfile/] to the root "
```

Comment:
Should link to the ci.jenkins.io index page

Reply:
You mean the "Jenkins CI Infrastructure" part of the string?

Reply:
yes